src/main/resources/system-user-permissions.txt

```diff
@@ -31,3 +31,6 @@ organizations.organizations.collection.get
 usergroups.collection.get
 users.collection.get
 users.item.get
+inventory-storage.classification-types.collection.get
+inventory-storage.instance-date-types.collection.get
```

Comment:
Added this now only, since we will be needing it after Matt merges his PR for instance date Object